metacat-common/src/main/java/com/netflix/metacat/common/QualifiedName.java

```diff
@@ -45,10 +45,7 @@
     private final String partitionName;
     private final String tableName;
     private final String viewName;
-    private String qualifiedName;
```

Comment:
Why do you not want to store this?

Reply:
unfortunately, this field is not populated in a thread-safe manner.
concurrency is not given any thought while populating this field.

Reply:
Agreed. We should have these fields and populate these in the constructor.

Reply:
is this a known performance bottleneck ?

Reply:
No. Since this class is immutable, It is better to initialize these in the constructor. What do you think?

Reply:
If its not a perf bottleneck, I would leave it alone..
otherwise, we would start creating the string rep for all immutable classes..

lets avoid pre-mature optimizations..